index.html

```diff
@@ -0,0 +1,141 @@
+            <section class="nav2 bg-white overflow-hidden position-relative py-2 d-lg-block d-none">
+                <ul class="d-flex align-items-center justify-content-between list-unstyled">
+                    <li><a href="#">U.S.</a></li>
+                    <li><a href="#">World</a></li>
+                    <li><a href="#">Business</a></li>
+                    <li><a href="#">Tech & Science</a></li>
+                    <li><a href="#">Culture</a></li>
+                    <li><a href="#">Newsgeek</a></li>
+                    <li><a href="#">Sports</a></li>
+                    <li><a href="#">Health</a></li>
+                    <li><a href="#">The Debate</a></li>
+                    <li><a href="#">Experts</a></li>
+                    <li><a href="#">Vantage</a></li>
+                    <li class="search-btn d-flex flex-row align-items-center">
+                        <input type="search" placeholder="Search" class="border-0">
+                        <i class="fas fa-search" aria-hidden="true"></i>
+                    </li>
+                </ul>
+            </section>
+            <section class="nav-3 bg-white overflow-hidden position-relative pt-0 pb-2 d-lg-block d-none">
+                <ul class="list-unstyled d-flex flex-row font-weight-bold ">
+                    <li>TRENDING</li>
+                    <li><a href="#">JOE BIDEN</a></li>
+                    <li><a href="#">DONALD TRUMP</a></li>
+                    <li><a href="#">INAUGURATION</a></li>
+                    <li><a href="#">CORONA VIRUS</a></li>
+                    <li><a href="#">STIMULUS PACKAGE</a></li>
+                    <li><a href="#">VACCINE</a></li>
+                    <li><a href="#">CAPITOL RIOTS</a></li>                                                                                                                                                                                                                                                       
+                </ul>
+            </section>
+        </nav>
+    </header>
+</nav>
+
+
+    <footer class="page-footer">
+        
+                <div class="container-fluid d-flex flex-row p-1 text-white">
+                   
+                    <div class="p-2">
+                        <a href="#"><img src="assets/logo/footerlogo.png" alt="" style="height: 50px;"></a>
+                    </div>   
+                    <div class="p-2 text-secondary">© 2021 NEWSWEEK DIGITAL LLC</div>
+
+
+                    <div class="p-2 text-secondary">
+                        <a href="#"><i class="fab fa-facebook-f"></i></a>
+                        <a href="#"><i class="fab fa-twitter"></i></a>
+                        <a href="#"><i class="fab fa-instagram"></i></a>
+                        <a href="#"><i class="fab fa-tumblr"></i></a>
+                        <a href="#"><i class="fab fa-linkedin-in"></i></a>
+                    </div>
+            
+                </div>
```

Comment:
### Desktop
- [ ] The spacing in the first row of tour footer is not like the original.
- [ ] The links should also not have any text decoration. 

The original page 👇🏾 

<img width="1299" alt="Screen Shot 2021-01-25 at 8 04 53 PM" src="https://user-images.githubusercontent.com/48804258/105753206-98286400-5f48-11eb-8897-60df952b47b9.png">

Your clone 👇🏾 
<img width="1397" alt="Screen Shot 2021-01-25 at 8 05 17 PM" src="https://user-images.githubusercontent.com/48804258/105753243-a6768000-5f48-11eb-9dab-b5e24b0df29f.png">

### Mobile

- [ ] The copyright text and social media icons should wrap under the logo on the mobile screen size.
Your clone 👇🏾 
<img width="306" alt="Screen Shot 2021-01-25 at 8 27 17 PM" src="https://user-images.githubusercontent.com/48804258/105755549-b9d71a80-5f4b-11eb-9706-5b9b6bb210b9.png">
The original page 👇🏾 
<img width="427" alt="Screen Shot 2021-01-25 at 8 27 35 PM" src="https://user-images.githubusercontent.com/48804258/105755569-c3f91900-5f4b-11eb-87a4-2c765b69dd08.png">